vscode-dotnet-runtime-library/src/EventStream/EventStreamEvents.ts

```diff
@@ -20,14 +21,16 @@ export class DotnetAcquisitionStarted extends IEvent {
     public readonly eventName = 'DotnetAcquisitionStarted';
     public readonly type = EventType.DotnetAcquisitionStart;
 
-    constructor(public readonly installKey: string, public readonly startingVersion: string, public readonly requestingExtensionId = '') {
+    constructor(public readonly installKey: DotnetInstall, public readonly startingVersion: string, public readonly requestingExtensionId = '') {
         super();
     }
 
     public getProperties() {
-        return {AcquisitionInstallKey : this.installKey,
+        return {
+                ...InstallToStrings(this.installKey),
```

Comment:
Yes, it will. With that said, it will actually standardize the events a bit more and make it easier to query them. Unfortunately, it will mean the new data is not the same as the old data.

Reply:
Actually we can probably keep the old data format to prevent churn on the old data :thinking:

Reply:
Thank you for pointing this out :) To make our lives easier I decided to add back in and keep the pre-existing properties.